relay-event-normalization/src/event.rs

```diff
@@ -2244,25 +2246,28 @@ mod tests {
             &mut event,
             &NormalizationConfig {
                 ai_model_costs: Some(&ModelCosts {
-                    version: 1,
-                    costs: vec![
-                        ModelCost {
-                            model_id: LazyGlob::new("claude-2*"),
-                            for_completion: false,
-                            cost_per_1k_tokens: 1.0,
-                        },
-                        ModelCost {
-                            model_id: LazyGlob::new("gpt4-21*"),
-                            for_completion: false,
-                            cost_per_1k_tokens: 2.0,
-                        },
-                        ModelCost {
-                            model_id: LazyGlob::new("gpt4-21*"),
-                            for_completion: true,
-                            cost_per_1k_tokens: 20.0,
-                        },
-                    ],
-                    models: HashMap::new(),
+                    version: 2,
+                    costs: vec![],
+                    models: HashMap::from([
+                        (
+                            "claude-2.1".to_owned(),
+                            ModelCostV2 {
+                                input_per_token: 0.01,
+                                output_per_token: 0.02,
```

Comment:
These values are off by a factor of 10 from the original test (1 per 1k tokens vs 10/20 per 1k tokens), is that intentional?